components/pedigree/resources/src/main/resources/pedigree/view/nodeMenu.js

```diff
@@ -872,7 +872,7 @@ define([
             var toDisplayedLateralityMap = {"bi" : "Bilateral", "u" : "Unilateral", "r" : "Right", "l" : "Left"};
 
             var toStoredAgeFx = function(value) {
-                if (isNaN(parseInt(value))) {
+                if (value && isNaN(parseInt(value))) {
```

Comment:
if `!value` the `else` branch will raise an exception

Reply:
Right, fixed.